app/views/events/_form.html.erb

```diff
@@ -33,6 +33,8 @@
     <div class="col-lg-10">
       <%= f.text_field :max_participants, :class => 'form-control' %>
       <%=f.error_span(:max_participants) %>
+      <%= f.label :participants_are_unlimited %>
```

Comment:
Unfortunately 0 is not grater than 0 and I would not change the validations for that hack. I think an extra (boolean) value is not that bad and normally it will be encapsulated through the getter and setter for max_participant.